proposals/nnnn-actors.md

```diff
@@ -663,7 +707,35 @@ When a given declaration (the "witness") satisfies a protocol requirement (the "
 * The requirement is `async`, or
 * the witness is non-isolated.
 
+An actor can satisfy an asynchronous requirement because any uses of the requirement are asynchronous, and can therefore suspend until the actor is available to execute them. Note that an actor can satisfy and asynchronous requirement with a synchronous one, in which case the normal notion of asynchronously accessing a synchronous declaration on an actor applies. For example:
+
+```swift
+protocol Server {
+  func send<Message: MessageType>(message: Message) async throws -> Message.Reply
+}
+
+actor MyServer : Server {
+  func send<Message: MessageType>(message: Message) throws -> Message.Reply { ... }  // okay, asynchronously accessed from clients of the protocol
+}
+```
```

Comment:
Ah, great to clarify this -- thanks, I was not sure myself if this would be ok or not.

I agree this is nice 👍 And of course it can also be actually async if it needs to... I like this flexibility 👍